### ISSUE TITLE
Accessibility check: falsche Bewertung durch Aria F96

### ISSUE THREAD

**Issue by fermiongithub** [AlbertWiersch/html-validator#16]
Testcase:

```
      <button aria-label="Navigation zeigen" popovertarget="navigation">
        ☰
      </button>
```

Result:

> [CHV] Error number 1 in line 14:27:
> Mismatch of visible button text and accessible name supplied via
> "aria-label" [A, 2.5.3]. Ensure that the button text ("☰")
> sufficiently matches the accessible name supplied via "aria-label"
> ("Navigation zeigen"). Visit
> https://www.w3.org/WAI/WCAG21/Techniques/failures/F96 for more
> information.

The result ist not correct.

Correct is:

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14
This link is mentioned at 
https://www.w3.org/WAI/WCAG21/Techniques/failures/F96

The symbol "☰" cannot be used as text.

**Comment by AlbertWiersch** [comment]
Thank you. The logic has been improved so that error should no longer be generated for the button text ("☰") in CSS HTML Validator 2027 BETA 2 (v26.4902), which I hope to release soon.

**Comment by fermiongithub** [comment]
Thanks : )

**Comment by AlbertWiersch** [comment]
You're welcome! And **CSS HTML Validator 2027 BETA 2 (v26.4902)** is now available for download. Please try it and let me know how it works.
https://www.htmlvalidator.com/download/registered.php?v=2027
